include/ccapi_cpp/service/ccapi_execution_management_service_bitfinex.h

```diff
@@ -227,10 +227,14 @@ class ExecutionManagementServiceBitfinex : public ExecutionManagementService {
   }
   void extractOrderInfoFromRequest(std::vector<Element>& elementList, const Request& request, const Request::Operation operation,
                                    const rj::Document& document) override {
-    if (operation == Request::Operation::CREATE_ORDER || operation == Request::Operation::CANCEL_ORDER) {
+    if (operation == Request::Operation::CREATE_ORDER) {
       Element element;
       this->extractOrderInfo(element, document[4][0]);
       elementList.emplace_back(std::move(element));
+    } else if (operation == Request::Operation::CANCEL_ORDER) {
+      Element element;
+      this->extractOrderInfo(element, document[4]);
+      elementList.emplace_back(std::move(element));
```

Comment:
Here are the demo responses from bitfinex for explanation:

submit order (aka CREATE_ORDER):
[
  MTS, 
  TYPE, 
  MESSAGE_ID, 
  null,
  
  [[
     ID,
     GID,
     CID,
     SYMBOL,
     MTS_CREATE, 
     MTS_UPDATE, 
     AMOUNT, 
     AMOUNT_ORIG, 
     TYPE,
     TYPE_PREV,
     MTS_TIF,
     _PLACEHOLDER,
     FLAGS,
     ORDER_STATUS,
     _PLACEHOLDER,
     _PLACEHOLDER,
     PRICE,
     PRICE_AVG,
     PRICE_TRAILING,
     PRICE_AUX_LIMIT,
     _PLACEHOLDER,
     _PLACEHOLDER,
     _PLACEHOLDER,
     _PLACEHOLDER,
     HIDDEN, 
     PLACED_ID,
     _PLACEHOLDER,
     _PLACEHOLDER,
     ROUTING,
     _PLACEHOLDER,
     _PLACEHOLDER,
     META
   ],
   ]
   
  CODE, 
  STATUS, 
  TEXT
]
    
[1567590617.442,"on-req",null,null,[[30630788061,null,1567590617439,"tBTCUSD",1567590617439,1567590617439,0.001,0.001,"LIMIT",null,null,null,0,"ACTIVE",null,null,15,0,0,0,null,null,null,0,null,null,null,null,"API>BFX",null,null,null]],null,"SUCCESS","Submitting 1 orders."]


cancel order:
[
  MTS, 
  TYPE, 
  MESSAGE_ID, 
  null,
  
  [
     ID,
     GID,
     CID,
     SYMBOL,
     MTS_CREATE, 
     MTS_UPDATE, 
     AMOUNT, 
     AMOUNT_ORIG, 
     TYPE,
     TYPE_PREV,
     MTS_TIF,
     _PLACEHOLDER,
     FLAGS,
     ORDER_STATUS,
     _PLACEHOLDER,
     _PLACEHOLDER,
     PRICE,
     PRICE_AVG,
     PRICE_TRAILING,
     PRICE_AUX_LIMIT,
     _PLACEHOLDER,
     _PLACEHOLDER,
     _PLACEHOLDER,
     _PLACEHOLDER,
     HIDDEN, 
     PLACED_ID,     
     _PLACEHOLDER,
     _PLACEHOLDER,
     ROUTING,
     _PLACEHOLDER,
     _PLACEHOLDER,
     META
   ]
   
  CODE, 
  STATUS, 
  TEXT
]
    
[1568298355648,"oc-req",null,null,[30937950333,null,1568298279766,"tBTCUSD",1568298281000,1568298281000,0.001,0.001,"LIMIT",null,null,null,0,"ACTIVE",null,null,15,0,0,0,null,null,null,0,0,null,null,null,"API>BFX",null,null,null],null,"SUCCESS","Submitted for cancellation; waiting for confirmation (ID: 30937950333)."]